cypress/integration/pages/auth/index.spec.ts

```diff
@@ -1,36 +1,105 @@
+    cy.get('.registration-form__text-input').eq(3).type('TestPassword');
+    cy.get('.registration-form__text-input').eq(4).type('TestPassword');
+    cy.get('.registration-form__text-input').eq(5).type('TestAethery@crit.io');
 
-    cy.get('.registration-form__button').first().click()
+    cy.get('.registration-form__button').first().click();
 
-    cy.get('.registration-form__success-message').should('be.visible')
-  })
+    cy.get('.registration-form__success-message').should('be.visible');
+  });
 
   it('should print out errors', () => {
-    cy.get('.registration-form__button').first().click()
+    cy.get('.registration-form__button').first().click();
 
-    cy.get('.registration-form__error-message').should('be.visible')
-  })
-})
+    cy.get('.registration-form__error-message').should('be.visible');
+  });
+});
 
-export default {}
+describe('Sign in page', () => {
+  beforeEach(() => {
+    cy.visit('/auth/sign-in');
+  });
+
+  it('should navigate to the Sign in page', () => {
+    cy.url().should('include', '/auth/sign-in');
+  });
+
+  it('should find form in the sign in page', () => {
+    cy.get('.login-form').contains('p', 'Not Signed up?');
+  });
+
+  it('should be able to fill input fields and redirect the user if success', () => {
+    cy.intercept('POST', '**/api/users/login-user', {
+      body: 'access_token: 1234',
+      statusCode: 201,
+    });
+
+    cy.get('.login-form__text-field').eq(0).type('testemail@crit.io');
+    cy.get('.login-form__text-field').eq(1).type('testpassword');
+
+    cy.get('.login-form__submit').first().click();
+    cy.url().should('include', '/home');
+  });
+
+  it('should show an error message if email does not exist', () => {
+    cy.intercept('POST', '**/api/users/login-user', {
+      body: {
+        message: {
+          message: 'testEmail@email.io does not exist',
+        },
+      },
+      statusCode: 400,
+    });
+
+    cy.get('.login-form__text-field').eq(0).type('testEmail@email.io');
+    cy.get('.login-form__text-field').eq(1).type('testpassword');
+
+    cy.get('.login-form__submit').first().click();
+
+    cy.get('.login-form__error-message').should('be.visible');
+  });
+
+  it('should show an error message if password is not accepted', () => {
+    cy.intercept('POST', '**/api/users/login-user', {
+      body: 'unauthorized',
+      statusCode: 401,
+    });
+
+    cy.get('.login-form__text-field').eq(0).type('testemail@crit.io');
+    cy.get('.login-form__text-field').eq(1).type('testpasswordd');
+
+    cy.get('.login-form__submit').first().click();
+
+    cy.get('.login-form__error-message').should('be.visible');
+  });
+
+  it('should print out errors if fields are not filled', () => {
+    cy.get('.login-form__submit').first().click();
+
+    cy.get('.login-form__error-message').should('be.visible');
+  });
+});
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {};
```

Comment:
Unfortunately, Typescript doesn't like it, here's a link for a better [explanation](https://stackoverflow.com/a/56577324/16299489) for it 

![image](https://user-images.githubusercontent.com/28055566/174491554-db6ac6cc-1217-4362-ba12-100bfe3b12a1.png)

I haven't really found a fix for it without the export default

Reply:
Hm, since we don't need to merge any definitions from the cypress test files, we can exclude the cypress folder from the `.tsconfig` 

Otherwise we would have to add this in every cypress test which serves no real purpose 🤔